src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -51,17 +56,46 @@
     public static final String TAG_DESC_FRIEND = " " + PREFIX_TAG + VALID_TAG_FRIEND;
     public static final String TAG_DESC_HUSBAND = " " + PREFIX_TAG + VALID_TAG_HUSBAND;
 
+    public static final String VALID_MEETING_NAME_CS2103 = "CS2103 Project";
+    public static final String VALID_MEETING_NAME_NUSSU = "Nussu meeting";
+    public static final String VALID_DATE_CS2103 = "11-04-2022";
+    public static final String VALID_DATE_NUSSU = "22-04-2022";
+    public static final String VALID_START_TIME_CS2103 = "1630";
+    public static final String VALID_START_TIME_NUSSU = "1530";
+    public static final String VALID_END_TIME_CS2103 = "1800";
+    public static final String VALID_END_TIME_NUSSU = "1730";
+    public static final String VALID_TAG_CS2103 = "V1point4";
+    public static final String VALID_TAG_NUSSU = "important";
+
+    public static final String MEETING_NAME_DESC_CS2103 = " " + PREFIX_MEETING_NAME + VALID_MEETING_NAME_CS2103;
+    public static final String MEETING_NAME_DESC_NUSSU = " " + PREFIX_MEETING_NAME + VALID_MEETING_NAME_NUSSU;
+    public static final String DATE_DESC_CS2103 = " " + PREFIX_DATE + VALID_DATE_CS2103;
+    public static final String DATE_DESC_NUSSU = " " + PREFIX_DATE + VALID_DATE_NUSSU;
+    public static final String START_TIME_DESC_CS2103 = " " + PREFIX_START_TIME + VALID_START_TIME_CS2103;
+    public static final String START_TIME_DESC_NUSSU = " " + PREFIX_START_TIME + VALID_START_TIME_NUSSU;
+    public static final String END_TIME_DESC_CS2103 = " " + PREFIX_END_TIME + VALID_END_TIME_CS2103;
+    public static final String END_TIME_DESC_NUSSU = " " + PREFIX_END_TIME + VALID_END_TIME_NUSSU;
+    public static final String TAG_DESC_CS2103 = " " + PREFIX_TAG + VALID_TAG_CS2103;
+    public static final String TAG_DESC_NUSSU = " " + PREFIX_TAG + VALID_TAG_NUSSU;
+
     public static final String INVALID_NAME_DESC = " " + PREFIX_NAME + "James&"; // '&' not allowed in names
     public static final String INVALID_PHONE_DESC = " " + PREFIX_PHONE + "911a"; // 'a' not allowed in phones
     public static final String INVALID_EMAIL_DESC = " " + PREFIX_EMAIL + "bob!yahoo"; // missing '@' symbol
     public static final String INVALID_ADDRESS_DESC = " " + PREFIX_TELEGRAM + "#"; // empty string not allowed
     public static final String INVALID_TAG_DESC = " " + PREFIX_TAG + "hubby/husband"; // '/' not allowed in tags
+    public static final String INVALID_MEETING_NAME_DESC =
+            " " + PREFIX_MEETING_NAME + "&"; // '&' not allowed in meeting names
+    public static final String INVALID_DATE_DESC = " " + PREFIX_DATE + "09-11-2022aB"; // 'ab' not allowed in dates
+    public static final String INVALID_START_TIME_DESC = " " + PREFIX_START_TIME + ":"; // empty string not allowed
```

Comment:
":" isn't exactly an empty string, so we should probably update the comment to be more accurate